### PR TITLE
Release 0.10.0

### PR DESCRIPTION
**Pull request by k0ste**
* [FEATURE] Add device include/exclude filters for the automatic scanning #99
* [ENHANCEMENT] Critical metrics for SCSI disks added #131
* [CHANGE] Remove duplicate smartctl_device_status metric #137
* [CHANGE] Fix reported Data bytes Read/Written on SSDs #138
* [FEATURE] Add background scanning for devices #140
* [ENHANCEMENT] Added device name to logger rc code parser #141

### CODE REVIEW

**Comment by k0ste** [comment]
@NiceGuyIT 🙏

**Comment by NiceGuyIT** [comment]
@k0ste I was going to see if we can include PR #138 before doing a release.

**Comment by frittentheke** [comment]
> @k0ste I was going to see if we can include PR #138 before doing a release.

@NiceGuyIT  ... what about https://github.com/prometheus-community/smartctl_exporter/pull/137 ... provided it's valid, it seems like a small cleanup as well.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 0.9.2 / 2023-06-10
```
Comment (SuperQ) [inline CHANGELOG.md:1]: Patch releases are for `[BUGFIX]` only.

```suggestion
## 0.10.0 / 2023-06-10
```

**Review by SuperQ — CHANGES_REQUESTED** [review]

**Comment by k0ste** [comment]
> @k0ste Can you make the requested changes and add the other merged PRs?

Done ✅

**Comment by NiceGuyIT** [comment]
@SuperQ Do I merge this release PR and then `git tag` the master branch with the new VERSION to kick off the build process? Is there anything I should watch out for or pay attention to during the release process?

**Comment by SuperQ** [comment]
@NiceGuyIT Yup, that's all you need to do. Checkout the `tag-release` git config alias in the Prometheus RELEASES docs.

The only thing to check is that the build publishes the releases to draft and contains the release artifacts.

**Comment by NiceGuyIT** [comment]
Thanks @SuperQ! Release 0.10.0 is out. Now I can help with the releases going forward.

**Pull request merged**